### DISCUSSION TITLE
change matching text highlight color

### DISCUSSION THREAD

**Discussion by Roadhammer-Gaming** [phcode-dev/phoenix#1671] [Ideas]
I've noticed in dark mode when clicking on <div> for example, the highlight color that identifies the start <div> and the ending </div> is almost the same color as the dark background, is there a way that can be changed in options somewhere?
![darkhighlightexample](https://github.com/phcode-dev/phoenix/assets/160162199/03d86b0a-2bd5-4ea5-b26b-f925b94ee41e)

**Comment by abose** [comment]
Currently, only creating a new theme can change the colors and its not possible to change single colors. We we mostly be adding this feature as in integrated theme editor.
